src/backend/src/prisma/schema.prisma

```diff
@@ -927,3 +929,18 @@ model Milestone {
   organizationId String
   organization   Organization @relation(fields: [organizationId], references: [organizationId])
 }
+
+model Announcement {
```

Comment:
There is no text in this model. This should also be a many-to-many relation, so you need a relation to be specified